.gitlab/binary_build/linux.yml

```
@@ -14,7 +14,27 @@ build_dogstatsd_static-deb_x64:
     - inv -e dogstatsd.build --static --major-version 7
     - $S3_CP_CMD $SRC_PATH/$STATIC_BINARIES_DIR/dogstatsd $S3_ARTIFACTS_URI/static/dogstatsd
 
-build_dogstatsd-deb_x64:
+build_dogstatsd_static-binary_arm64:
+  stage: binary_build
+  rules:
+    !reference [.on_a7]
```

We only run arm64 builds when the `ALL_BUILDS` option is set, so I think this should also apply to these new jobs.

```suggestion
    !reference [.on_all_builds_a7]
```